IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TrackSlabGeneral/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_TrackSlabGeneral" UniqueId="842a5e4e-a131-ea5d-4ff7-1176fbb44a3c" ApplicableType="IfcSlab/TRACKSLAB,IfcSlab/BASESLAB" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="TechnicalStandard_2MQ1oBlHFBhUArnmHLc4W" />
+		<DocProperty xsi:nil="true" href="TrackEdgeID_1SbfmW4bP5EOIA2WOcWPHo" />
+	</Properties>
+</DocPropertySet>
+
```

Comment:
A question: Is it correct to include TrackEdgeID at this level? Is it a reference to a network edge? Could this be better handled at a more aggregated level such as the spatial breakdown?